Pipfile

```diff
@@ -10,16 +10,17 @@ twine = "*"
 pytest-xdist = "*"
 invoke = "*"
 crayons = "*"
-pythonfinder = {path = ".", editable = true}
+pythonfinder = {path = ".",editable = true}
 sphinx-rtd-theme = "*"
 rope = "*"
-black = {version = "*", markers = "python_version>='3.6'"}
+black = {version = "*",markers = "python_version>='3.6'"}
 parver = "*"
 towncrier = "*"
-mypy = {version = "*", markers = "python_version>='3.4'"}
+mypy = {version = "*",markers = "python_version>='3.4'"}
 mypy_extensions = "*"
-typed-ast = {version = "*", markers = "python_version>='3.3'"}
+typed-ast = {version = "*",markers = "python_version>='3.3'"}
```

Comment:
The spaces are missing?

Reply:
yep, that happens anytime i use pipenv lately... it's really annoying... would love if that one gets fixed :)

Reply:
This is weired. I've never met this before.